plugins/premium/mentions.md

```diff
@@ -281,16 +280,18 @@ tinymce.init({
         '</div>'
       );
       success(div);
+      // Or simply use the predefined 'profile' template.
+      // success({ type: 'profile', user: userDetail });
     });
   }
 });
 ```
 
-#### `mentions_select` `profile`
+#### `mentions_select` with predefined templates
 
 {{site.requires_5_6v}}
 
-If the option is set to "profile", the mentions `profile` template will be used. For details on the user properties required for the "profile" setting, see: [User properties](#userproperties).
+If `mentions_select` is resolved with an object specifying type and user details, a predefined select template will be used. The only valid template at this time is `'profile'`. For details on the user properties required for the `profile` template, see: [User properties](#userproperties).
```

Comment:
Saying `at this time` is generally discouraged, as it can imply we are going to add additional types when we may not. It's also hard to tell when "at this time" was referring to.

I can't think of anything better, so tagging @tylerkelly13 as he knows all :stuck_out_tongue:

Reply:
```suggestion
If `mentions_select` is resolved with an object specifying the type and user details, a predefined select template will be used. To use the predefined template, set `mentions_select` to `'profile'`. For details on the user properties required for the `profile` template, see: [User properties](#userproperties).
```

Reply:
My suggestion may or may not be better. we say "A predefined template", implying one or more. so stating how to use the only one should be good. not sure if that "the"  is correct, please correct if not.

Reply:
I reworded it to say "the" so as to not imply that there'll be more templates added in the future. `mentions_select` can no longer be set to `'profile'` so I refered to an example to clarify the usage.